samples/local_attestation/common/attestation.cpp

```diff
@@ -21,36 +28,33 @@ bool Attestation::generate_local_report(
     const uint8_t* data,
     const size_t data_size,
     uint8_t** report_buf,
-    size_t* remote_report_buf_size)
+    size_t* local_report_buf_size)
 {
     bool ret = false;
     uint8_t sha256[32];
-    oe_result_t result = OE_OK;
-    uint8_t* temp_buf = NULL;
+    oe_result_t result = OE_OK;    
 
     if (m_crypto->Sha256(data, data_size, sha256) != 0)
     {
         goto exit;
     }
 
+    // Initialize attester and use the SGX plugin.
+    oe_attester_initialize();
+
+    oe_attester_select_format(&sgx_local_uuid, 1, &selected_format);
+
     // To generate a local report that just needs to be attested by another
```

Comment:
Suggest to update the comment as there is no local report anymore.

Reply:
Fixed.